crates/dslab-storage/src/disk.rs

```diff
@@ -136,21 +138,49 @@ impl DiskBuilder {
         self
     }
 
+    /// Sets read concurrent operations limit.
```

Comment:
Правильнее написать так

```suggestion
    /// Sets concurrent read operations limit.
```

и переименовать соответственно поля и методы, перенеся слово `concurrent` в начало.